views/aggregate-flow-metadata.json

```diff
@@ -312,12 +331,18 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-census",
-            "description": {
-                "summary": "Waits for the Census sync run to finish running."
-            }
+            "path_containing_flow": "prefect_census/syncs.py",
```

Comment:
What's the difference between this an entrypoint?

Reply:
both are required by POST deployment endpoint, both were requested by UI